### DISCUSSION TITLE
Extending Queries

### DISCUSSION THREAD

**Discussion by smith-isaac** [nvim-treesitter/nvim-treesitter#3473] [Q&A] [answered]
The README says that extending queries can be easily done by adding a query file to `after/queries/<language>/` folder based on the runtime. I have been trying to add some additional queries for a couple languages, namely C and Julia, but I am having trouble getting the queries in the `after` directory to get used. Am I missing something? Here is an example.

I have placed
```scm
(string_literal) @spell
```
into `~/.config/nvim/after/queries/c/highlights.scm` but this query is not being used at all. Do I need to do something else to get this query sourced somehow?

I have been working around this by manually editing the highlights.scm in the plugin files, but I would like to move away from forking for such small changes.

**Comment by clason** [comment, marked as answer]
The readme is outdated.

You need to put `; extends` at the top of the query now to actually extend rather than replace (or be replaced).

**Reply by smith-isaac** [reply]
Works great. Thanks!
